src/App.vue

```diff
@@ -35,5 +36,28 @@ export default {
   .container {
     max-width: 720px;
   }
+  .v-toolbar {
+    &__title {
+      a {
+        text-decoration: none;
+        display: flex;
+        align-items: center;
+        font-size: 2em;
+        color: #000;
```

Comment:
I would add a `font-family: CircularStd-Black, Circular Std`, you might then need to bump the `font-size: 4em`

Reply:
@ryanSN Done!